### PR TITLE
fix for CKV_GCP_50 - MySQL test only target mySQL

### PR DESCRIPTION
**Pull request by JamesWoolfenden**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/terraform/checks/resource/gcp/example_CloudMySqlLocalInfileOff/main.tf [hunk 1]
```diff
@@ -0,0 +1,123 @@
+
+                resource "google_sql_database_instance" "fail" {
```
Comment (gruebel) [inline tests/terraform/checks/resource/gcp/example_CloudMySqlLocalInfileOff/main.tf:2]: @JamesWoolfenden could align the TF code here, it looks super strange 😄

**Review by gruebel — COMMENTED** [review]
please just the format for the TF file, then it is good to merge

File: tests/terraform/checks/resource/gcp/example_CloudMySqlLocalInfileOff/main.tf (hunk 1 above)
Comment (JamesWoolfenden) [reply tests/terraform/checks/resource/gcp/example_CloudMySqlLocalInfileOff/main.tf]: done

**Review by gruebel — APPROVED** [review]
💪

**Pull request merged**